packages/flame/lib/src/game/base_game.dart

```diff
@@ -24,33 +23,20 @@ import 'viewport.dart';
 /// This is the recommended structure to use for most games.
 /// It is based on the Component system.
 class BaseGame extends Game {
+  BaseGame() {
+    components = createComponentSet();
+    _cameraComponent = CameraWrapper(Camera(), components);
```

Comment:
Should this property not reflect the class name for the time being, even if it is internal?

As far as I understood is that when #906 gets merged this property wont exist anyway?